universal-application-tool-0.0.1/app/views/dev/AwsViewStorageStrategy.java

```diff
@@ -0,0 +1,58 @@
+package views.dev;
```

Comment:
suggestion (blocking): move out of `dev` package. `dev` should only be for classes that are used in the development environment.